crates/version/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "calimero-version"
+version = "0.5.0"
+edition = "2021"
+
```

Comment:
as mentioned, follow the template in other crates, there are some missing fields here

<img src="https://github.com/user-attachments/assets/a8c35afc-3d9a-4d05-b663-4f458e6b3b04" width="50%" />